mars_rover/models/curiosity_path/model.sdf

```diff
@@ -17,7 +17,7 @@
         <collision name='collision'>
           <geometry>
             <mesh>
-              <uri>/root/src/spaceros_demo_ws/install/mars_rover/share/mars_rover/models/curiosity_path/meshes/curiosity_path.stl</uri>
+              <uri>/home/spaceros-user/src/spaceros_demo/install/mars_rover/share/mars_rover/models/curiosity_path/meshes/curiosity_path.stl</uri>
```

Comment:
Mostly asking out of curiosity, I don't consider this a blocking change or even a nitpick but why drop the `_ws`?